### PR TITLE
query stats on connect

### PR DESCRIPTION
**Pull request by fippo**
with a longer interval it would be good to have the stats on connect so we can determine firstcandidatepairtype even if the period getStats doesn't apply

### CODE REVIEW

File: rtcstats.js [hunk 1]
```diff
@@ -171,31 +171,39 @@ module.exports = function(trace, getStatsInterval, prefixesToWrap) {
       // TODO: do we want one big interval and all peerconnections
       //    queried in that or one setInterval per PC?
       //    we have to collect results anyway so...
+      var getStats = function() {
+        if (isFirefox || isSafari) {
+          pc.getStats(null).then(function(res) {
+            var now = map2obj(res);
+            var base = JSON.parse(JSON.stringify(now)); // our new prev
+            trace('getstats', id, deltaCompression(prev, now));
```
Comment (ggarber) [inline rtcstats.js:177]: I thought it was `getStats` camelCase

File: rtcstats.js [hunk 2]
```diff
@@ -171,31 +171,39 @@ module.exports = function(trace, getStatsInterval, prefixesToWrap) {
...
+            prev = base;
+          });
+        } else {
+          pc.getStats(function(res) {
+            var now = mangleChromeStats(pc, res);
+            var base = JSON.parse(JSON.stringify(now)); // our new prev
```
Comment (ggarber) [inline rtcstats.js:183]: These 3 lines are duplicate in case you want to refactor them

**Review by ggarber — APPROVED** [review]

File: rtcstats.js (hunk 2 above)
Comment (fippo) [reply rtcstats.js:183]: the refactoring that needs to be done here is to get rid of the legacy stats

File: rtcstats.js (hunk 1 above)
Comment (fippo) [reply rtcstats.js:177]: see the server, getStats and getstats have very different semantics

**Pull request merged**